packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -2681,8 +2681,8 @@ describe('KeyringController', () => {
               },
               async ({ controller, encryptor }) => {
                 jest.spyOn(encryptor, 'isVaultUpdated').mockReturnValue(false);
-                const encryptSpy = jest.spyOn(encryptor, 'encryptWithDetail');
-                jest.spyOn(encryptor, 'decryptWithKey').mockResolvedValueOnce([
+                const encryptSpy = jest.spyOn(encryptor, 'encrypt');
```

Comment:
What is the intent behind this change? The test description seems to indicate that an encryption key is being used, but it's not after this change

Reply:
Oh I see, understood, thanks